01-getting-started.Rmd

```diff
@@ -326,11 +326,11 @@ flights
 
 Let's unpack this output:
 
-* `A tibble: 336,776 x 19`: A `tibble` is a specific kind of data frame in R.\index{tibble}  This particular data frame has
-    + `336,776` rows corresponding to different *observations*. Here, each observation is a flight.
-    + `19` columns corresponding to 19 *variables* describing each observation.
+* ``A tibble: `r comma(nrow(nycflights13::flights))` x `r ncol(nycflights13::flights)` ``: A `tibble` is a specific kind of data frame in R.\index{tibble}  This particular data frame has
```

Comment:
Let's remove the `nycflights13::` headers. I feel they make the code unnecessarily long.

Reply:
Also, could you use the pipe instead of nested parentheses? e.g. `flights %>% nrow() %>% comma()`

Reply:
These comments apply to rest of PR